modules/understanding-the-administration-event-details.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// *operating/using-the-administration-events-dashboard.adoc
+
+:_content-type: CONCEPT
+[id="understanding-the-administration-event-details_{context}"]
+= Understanding the administration event details
+
+The administration event details page organizes information in the following groups:
+
+* *Error*: Provides log information that explains the nature of the error or event. It provides important information for understanding the specific issue or incident. This field contains information about the context of the event and the steps you need to take to fix the error.
```

Comment:
While this today is shown as an "Error" or "Warn" (depending on the level of the administration event, it really is the "message" field in the administration event API.

Not sure how detailed we have to be here though, just as a note.

Especially, since the UI is merging the "message" with the "hint" fields, which essentially gives you the following:
- The message which lead to the creation of the event
- Detailed information about the issue with potential remediation steps for the customer